openslides/assignments/static/js/assignments/site.js

```diff
@@ -459,8 +413,9 @@ angular.module('OpenSlidesApp.assignments.site', [
     'PdfMakeBallotPaperProvider',
     'gettextCatalog',
     function($scope, $http, $filter, filterFilter, gettext, ngDialog, AssignmentForm, operator, Assignment,
-        User, assignment, phases, Projector, ProjectionDefault, AssignmentContentProvider, BallotContentProvider,
+        User, assignmentId, phases, Projector, ProjectionDefault, AssignmentContentProvider, BallotContentProvider,
         PdfMakeDocumentProvider, PdfMakeBallotPaperProvider, gettextCatalog) {
+        var assignment = Assignment.get(assignmentId);
```

Comment:
Question: Quite some times I changed the injection in the way that not the object but the id is injected. In this (and more) cases the controller needs the object and does not only bind the object. So I `get` the object in the controller. Is this acceptable or should we inject the object:
```
resolve: {
    assignment: ['$statePrams', 'Assignment', function ($stateParams, Assignement) {
        return Assignment.get($stateParams.id);
    }]
}
```
so the commented line could be removed?